lametro/static/css/city_custom.css

```diff
@@ -58,6 +58,20 @@ h1, .h1 {
     padding-bottom: 20px;
 }
 
+#section-events h2 {
+    margin-top: 20px !important;
+    margin-bottom: 0px !important;
+}
+
+#upcoming-meetings {
+    margin-bottom: 20px;
+}
+
+#past-meetings {
+    border-top: 1px solid #ccc;
```

Comment:
Can you say more about doing this in CSS, instead of with an `<hr>` element?

Reply:
I initially tried to use <hr> but was having trouble getting it to show up. Turns out I was using bootstrap 5 classes instead of bootstrap 3 ones.